components/Hero/Hero.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "next/image";
+import { Link } from "components/Link/Link";
+import { CTA, Photo } from "types/layout-schema-types";
+import { urlFor } from "lib/client";
+
+export interface HeroSection extends React.HTMLProps<HTMLElement> {
+  cta: CTA;
+  description: string;
+  photo: Photo;
+  title: string;
+}
+
+export const Hero = ({ title, description, photo, cta }: HeroSection) => {
+  return (
+    <section className="py-10">
+      <div className="flex max-w-screen-xl flex-col lg:mx-auto lg:flex-row lg:items-center">
+        <article className="mb-14 lg:mb-0 lg:w-5/12">
+          <h1 className="mb-1 max-w-4xl text-blue md:mb-5">{title}</h1>
+          <p className="mb-8 font-inter text-base font-medium text-blue md:mb-9 md:text-xl">
+            {description}
+          </p>
+          <Link href={cta.url} appearance="button">
+            {cta.name}
+          </Link>
+        </article>
+        <figure className="overflow-hidden">
```

Comment:
> Usually a `<figure>` is an image, illustration, diagram, code snippet, etc., that is referenced in the main flow of a document, **but that can be moved to another part of the document or to an appendix without affecting the main flow**.

The last part, makes me think that this isn't the best element to represent the image.

Reply:
> The last part, makes me think that this isn't the best element to represent the image.

Then, what can I use instead?